nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/GetHDFSFileInfo.java

```diff
@@ -596,6 +658,23 @@ protected HDFSFileInfoRequest updateRequestDetails(ProcessContext context, Proce
         return req;
     }
 
+    static class ExecutionContext {
+        private int nrOfWaitingHDFSObjects;
+        private FlowFile flowfile;
+
+        public void reset() {
+            nrOfWaitingHDFSObjects = 0;
+            flowfile = null;
+        }
+
+        public void finish(ProcessSession session) {
```

Comment:
Minor: Using the same access modifier (package private is appropriate I think) for the fields and the methods would be more consistent because both the fields and the methods are accessed directly from the outer class.